### ISSUE TITLE
Oct 5 - Take a Closer Look at Visual Studio Code

### ISSUE THREAD

**Issue by mebaumb** [microsoft/ReactorPlanning#64]
# Event Information
| Category | Details |
|-----------|---------|
| Reactor Topic | Coding, Languages, and Frameworks |
| Monthly Topic (if applicable) | OSS |
| Weekly Topic (if applicable) | VS Code |
| Stream Date | Oct  5, 2020 | 
| Stream Time | 4:00pm PT | 
| Stream Duration | 60 | 
| Title | Take a Closer Look at Visual Studio Code |
| Description |<p>Learn what it's like to work on Visual Studio Code: how a project is built, some top features, and how to start coding with the editor. Join Visual Studio Code Program Manager Sana Ajani and Jackson Kearl, Software Engineer, for an insider’s look at this powerful code editor.</p><p>Prerequisites: This session recommends a basic understanding of code editors. If you would like to follow along, download Visual Studio Code (https://code.visualstudio.com/).</p>  |
| Speaker | Sana Ajani, Jackson Kearl and Meaghan Lewis |
| Moderator | none |
| SharePoint ID | 8000 |  

Additional Resources:  
<Additional resources that can be linked from social media for promotion or added to the description of VOD>

# Event Checklist

## 10 Weeks Prior
- [x] Reactor Content Team (RCT) or Speaker opens a new issue for an upcoming stream filling in the Event Information table. Both RCT and Speaker are marked as assignees.
- [x] RCT and Speaker discuss details of the event in the comments of this issue. RCT adds a _confirmed-event_ label and _global-twitch-event_ tag to the issue and adds event to the __Global Live Streams__ project. Issue will appear in the __Upcoming Streams__ column
- [x] Speaker adds a _ready-for-marketing-edit_ label so the RCT can review title and description

## 8 Weeks Prior
- [x] RCT does a review of title and description and removes the _ready-for-marketing-edit_ label 
- [x] RCT moves the issue to the __Ready for SharePoint__ column
- [x] RCT informs the Reactor Marketing Team of the new event information
- [x] RCT provides speaker with assets (e.g. slides, overlays) and the survey data for the event
- [x] RCT moves the issue to the __MONTH Streams Prep__ column (according to the month it will be streamed in)

## 2 Weeks Prior *(Done in /microsoft/Reactors)*
- [ ] Speaker opens a PR with the new content for the stream on the [customer-facing Reactor repository](https://github.com/microsoft/Reactors) and adds the _content-development_ and _global-twitch-event_ tag to the PR. 
- [ ] Speaker adds to the same PR a modification to the [Streaming README](https://github.com/microsoft/Reactors/tree/main/Online) to include their stream information.
- [ ] Speaker adds the RCT assigned to the issue as the reviewer when it is ready for review/merge
- [ ] RCT reviews, approves, and merges the changes

## 1 Week Prior
- [ ] Speaker and Moderator test setup for streaming (instructions coming soon)
- [ ] RCT checks in with the speaker and moves the issue to the __Ready to Stream__ column

## During and After Stream
- [ ] Speaker and Moderator run the stream
- [ ] RCT submits recording to be uploaded to the Reactor YouTube and to be made available on LearnTV
- [ ] RCT moves issue to the __Stream Complete__ column

**Comment by meaghanlewis** [comment]
@sana-ajani we put all details for our streams here in GitHub. Just wanted to let you know this issue exists. 😄

**Comment by mebaumb** [comment]
Hey @meaghanlewis @sana-ajani - is this ready for a marketing edit?

**Comment by meaghanlewis** [comment]
Not yet! I am confirming that Sana is still available for this stream.

**Comment by meaghanlewis** [comment]
@mebaumb @KimberlyBolton1 this is no ready for a marketing edit.

**Comment by KimberlyBolton1** [comment]
@meaghanlewis @sana-ajani edited abstract below. Are there any prereq's or tools recommended to attend? I've added a suggestion but please update it as needed.

**Take a Closer Look at Visual Studio Code**
Learn what it's like to work on Visual Studio Code: how a project is built, some top features, and how to start coding with the editor. Join Visual Studio Code Program Manager Sana Ajani for an insider’s look at this powerful code editor. 

Prerequisites: This session recommends a basic understanding of code editors. If you would like to follow along, download Visual Studio Code (https://code.visualstudio.com/).

**Comment by meaghanlewis** [comment]
This edit looks good to me! ✅

**Comment by KimberlyBolton1** [comment]
@mebaumb abstract final and moved to Ready for SharePoint.

**Comment by mebaumb** [comment]
This has been published! (9/2)
